bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -224,6 +224,22 @@
 		<state pattern="%s" readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="obstruction_alarm">
+		<item-type>Contact</item-type>
+		<label>Obstruction Alarm</label>
+		<description>Obstruction alarm of the blinds. The blinds were stopped and moved a bit in the opposite direction.</description>
+		<category>Blinds</category>
+		<state pattern="%s" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="temperature_alarm">
+		<item-type>Contact</item-type>
```

Comment:
The type choice looks strange to me. Shouldn't it be rather a "Switch" type, meaning there was an overheating of the motor (ON)or not (OFF) ?

Reply:
Done.